analysis/import/SCMLogParser/src/main/kotlin/de/maibornwolff/codecharta/importer/scmlogparser/input/metrics/NumberOfAuthors.kt

```diff
@@ -20,4 +21,8 @@ class NumberOfAuthors: Metric {
     override fun value(): Number {
         return authors.size
     }
+
+    override fun attributeType(): AttributeType {
+        return AttributeType.relative
```

Comment:
Should we set this to relative? This value neither does sense if sum it nor if we average/median it. This value would require to check for unique authors and sum them then.

example:
Folder A, contains File B and C.
A was commited by Alex and Ben at some point.
B was commited by Alex and Alexander at some point.

I would expect that when I hover over the folder, it would display that 3 authors worked on that package.

Reply:
I agree with you that neither aggregation mode is perfect. However, the behaviour you described (which I think makes sense) is beyond the scope of this PR/issue, and would need to be implemented as a new aggregation mode in the visualization.

So for now the question is: which aggregation mode is less wrong? Summing them up (in the current fashion) makes no sense obviously, while the median gives at least an idea on how many people worked on an average file within a package.

Therefore I suggest: Leave this relative for this PR & open a new issue for the better aggregation mode.

Reply:
Yea I agree with @alexhunziker ;)